documentation/release_notes.markdown

```diff
@@ -8,42 +8,75 @@ canonical: "/puppetdb/latest/release_notes.html"
 [kahadb_corruption]: ./trouble_kahadb_corruption.html
 [pg_trgm]: http://www.postgresql.org/docs/current/static/pgtrgm.html
 [upgrading]: ./api/query/v4/preparing-for-3.0.html
+[puppetdb-module]: https://forge.puppetlabs.com/puppetlabs/puppetdb
 
 3.0.0
 -----
 
-### Upgrading
-
-* For the best-possible performance and scaling capacity, we recommend
- PostgreSQL version 9.4 or newer with the [`pg_trgm`][pg_trgm]
- extension enabled, as explained [here][configure_postgres], and we
- have officially retired versions earlier than 9.4. HSQLDB is
- only recommended for local development because it has a number of
- scaling and operational issues.
+PuppetDB 3.0.0 is a major release which introduces some major new features,
+with some breaking changes.
 
-* Make sure that all of your PuppetDB instances are shut down, and
- only upgrade one at a time.
+### Contributors
 
-* Make sure to upgrade your puppetdb-terminus package (on the host
- where your Puppet Master lives), and restart your master service.
+Andrew Roetker, Andrii Nikitiuk, Chris Price, Eric Sorenson, John Duarte,
+Jarie Tappa, Ken Barber, Brian LaMattery, Lars Windolf, Mathieu Parent,
+Matthaus Owens, Nick Fagerlund, Rob Braden, Rob Browning, Russell Mull,
+Ryan Senior, Scott Garman, Wayne Warren, Wyatt Alt, Jean Bond and
+Darin Perusich.
```

Comment:
Nice catch! Indeed, it's Jorie. Also, it looks like maybe this list was alphabetized by first name at one point but a few names have fallen out of alphabetization, not a huge deal but could that get fixed?